src/main/java/org/betonquest/betonquest/compatibility/holograms/holographicdisplays/HolographicDisplaysHologram.java

```diff
@@ -144,4 +144,13 @@ public int size() {
     public void clear() {
         hologram.getLines().clear();
     }
+
+    /**
+     * Unable to change the range of visibility in HolographicDisplay
+     */
+    @Override
+    public void setMaxRange(final int index) {
+        // Don`t throw InstructionParseException because of calling this method in timer task
+        // *Log message with information about this is sent in HologramLoop#validateMaxRange
```

Comment:
No comments in the code. There are some special key words to solve PMD issues. But if you want to explain things, do it in the java docs.
```suggestion
        // Empty
```